### PR TITLE
Users update & contragents

### PR DESCRIPTION
**Pull request by ivanboika**
Added those models but haven't checked full app due to error described in discord channel

### CODE REVIEW

**Comment by ivanboika** [comment]
This pr gonna be pushed in main

File: api/src/models/contragent.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy import String, Float
+
+from ..database import Base
+from . import core_model
```
Comment (132dannys) [inline api/src/models/contragent.py:5]: from src.models.core_model import CoreModel

File: api/src/models/user.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy import String
 
-from ..database import Base, metadata
+from ..database import Base
+from . import core_model
 
 
-class User(SQLAlchemyBaseUserTableUUID, Base):
+class User(SQLAlchemyBaseUserTableUUID, Base, core_model.CoreModel):
```
Comment (132dannys) [inline api/src/models/user.py:9]: class SQLAlchemyBaseUserTableUUID alredy have uuid field

**Review by 132dannys — CHANGES_REQUESTED** [review]

File: api/src/models/user.py (hunk 2 above)
Comment (ivanboika) [reply api/src/models/user.py:9]: Fixed

File: api/src/models/contragent.py (hunk 1 above)
Comment (ivanboika) [reply api/src/models/contragent.py:5]: Deleted due to unused rn

File: api/src/models/contragent.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy import String, Float
+
+from ..database import Base
```
[the remark this answers is not in the document]
Comment (ivanboika) [reply api/src/models/contragent.py:4]: Done

**Pull request merged**